### PR TITLE
[TEVA 2126] add code for rerunning of cookie ab test

### PR DESCRIPTION
**Pull request by alexbowen**
https://dfedigital.atlassian.net/browse/TEVA-2126

### CODE REVIEW

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-3257.london.cloudapps.digital>

**Comment by milimaldedfe** [comment]
All looks good now. Thanks @alexbowen

**Review by cpjmcquillan — DISMISSED** [review]
The GDS variant should be using `govuk_cookie_banner` component

ok ill give it a go, would be nice if it worked as per their docs...

File: app/components/ab_test/cookies_banner_component/cookies_banner_component.html+gds.slim [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
         h2.govuk-cookie-banner__heading.govuk-heading-m Cookies on Teaching Vacancies
 
         .govuk-cookie-banner__content
           p We use some essential cookies to make this service work.
-          p We’d like to set additional cookies so we can remember your settings, understand how people use the service and make improvements.
+          p We’d also like to use analytics cookies so we can understand how you use the service and make improvements.
 
     .govuk-button-group
-      = govuk_button_to "Accept all cookies", create_path
+      = govuk_button_to "Accept analytics cookies", create_path, form_class: "govuk-button--primary"
```
Comment (alexbowen) [inline app/components/ab_test/cookies_banner_component/cookies_banner_component.html+gds.slim:12]: ill put these in locale files, should the gds variant win!

Comment (alexbowen) [inline app/components/ab_test/cookies_banner_component/cookies_banner_component.html+gds.slim:12]: as it probably should

Comment (alexbowen) [reply app/components/ab_test/cookies_banner_component/cookies_banner_component.html+gds.slim:12]: as it probably should

**Comment by david-mears-dfe** [comment]
How do we force a particular variant to show, again?

**Comment by david-mears-dfe** [comment]
So we're definitely using different text for the gds variant as compared to the other variants? Wondering if we should use the gds text across all variants, to make the test fairer - unless we actively want to test the effect of varying the text.

**Comment by stevenleggdfe** [comment]
Looking good - thanks @alexbowen !

**Comment by alexbowen** [comment]
> So we're definitely using different text for the gds variant as compared to the other variants? Wondering if we should use the gds text across all variants, to make the test fairer - unless we actively want to test the effect of varying the text.

yes i 

> So we're definitely using different text for the gds variant as compared to the other variants? Wondering if we should use the gds text across all variants, to make the test fairer - unless we actively want to test the effect of varying the text.

thats a question for product/content people really, they wont see it on PR

**Review by csutter — APPROVED** [review]

**Pull request merged**